ArcGISRuntimeSDKQt_CppSamples/AR/DisplayScenesInTabletopAR/android/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -0,0 +1,5 @@
+distributionBase=GRADLE_USER_HOME
+distributionPath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.6-bin.zip
```

Comment:
Is this a typo? I don't know much about gradle properties/scripts but this looks wrong.

Reply:
@JamesMBallard My knowledge is limited on this as well, but this was automatically generated by Qt Creator following the instructions from the toolkit [under the first step](https://github.com/Esri/arcgis-runtime-toolkit-qt/blob/master/Common/AR/README.md#configuration-for-android-devices), readme so I was operating under the assumption it was correct.

Reply:
Sounds good @DonMoJa. I see now it's probably escaping the `:`